docs/readmes/README.featureflags.md

```diff
@@ -48,3 +48,11 @@ their use cases.
 
 Aad-pod-identity has a new flag `immutable-user-msis` which can be used to prevent deletion of specified identities from VM/VMSS.
 The list is comma separated. Example: 00000000-0000-0000-0000-000000000000,11111111-1111-1111-1111-111111111111
+
+## Metadata header required flag
```

Comment:
Also, after this can you add `> Available from 1.6 release` similar to the rest of the features in the readme

Reply:
Done, thanks for your carefulness and patience